src/modules/AutoInstall.rb

```diff
@@ -301,6 +304,23 @@ def PXELocalBoot
       true
     end
 
+    # Implement pkgGpgCheck callback
+    #
+    # @param [Hash] data Output from `pkgGpgCheck` callback.
+    # @options data [String] "CheckPackageResult" Check result code according to libzypp.
+    # @options data [String] "Package" Package's name.
+    # @options data [String] "Localpath" Path to RPM file.
+    # @options data [String] "RepoMediaUrl" Media URL.
+    #   (it should match `media_url` key in AutoYaST profile).
+    # @return [String] "I" if the package should be accepted; otherwise
+    #   a blank string is returned (so no decision is made).
+    def pkg_gpg_check(data)
+      log.debug("pkgGpgCheck data: #{data}")
+      accept = PkgGpgCheckHandler.new(data, Profile.current).accept?
+      log.info("PkgGpgCheckerHandler for #{data["Package"]} returned #{accept}")
+      accept ? "I" : ""
```

Comment:
I'm just not sure here, maybe we should return "A" to explicitly abort, leaving the final decision on libzypp might lead to unexpected results...